# <mayhem_program::instructions::fee_recipients::FeeRecipient as anchor_lang::AccountDeserialize>::try_deserialize  (yours 1000 B at .text+0x31130; original 1000 B at .text+0x3cc58)
# distance 2 instructions; evidence: same-callees 7/7
--- original
+++ yours
@@ -48,7 +48,7 @@
 stxdw [r10-224], r1
 stw [r10-80], 3002
 stb [r10-152], 2
-stw [r10-208], 20
+stw [r10-208], 7
 stdw [r10-216], 58
 stdw [r10-232], 0
 mov64 r1, r10
@@ -83,7 +83,7 @@
 stdw [r8], 1
 stxdw [r6+16], r8
 stxdw [r6+8], r7
-stdw [r6], 1
+stb [r6], 1
 exit
 mov64 r4, 0
 jne r4, 0, +0x250
